src/common/translations/en.json

```diff
@@ -71,7 +72,9 @@
         "createWallet": "Create Basic Wallet",
         "createWalletNote": "Recommended for first-time users",
         "importWallet": "Import Existing Wallet",
-        "importWalletNote": "Restore your wallet using the recovery phrase"
+        "importWalletNote": "Restore your wallet using the recovery phrase",
+        "addReadOnlyWallet": "Add Read-only Wallet",
+        "addReadOnlyWalletNote": "Monitor assets without keys"
```

Comment:
```suggestion
        "addReadOnlyWalletNote": "Monitor assets without spending keys"
```

Reply:
@chrisli30 The reason to change to spending keys: https://twitter.com/matt_odell/status/1299298525570830341?s=19